mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule OpenApiSpex.Mixfile do
   use Mix.Project
 
-  @version "3.1.0"
+  @version "3.1.6"
```

Comment:
I'll migrate this back down to what it was or `3.1.1` before merge, I've just been having a _helluva_ time getting mix to update for some reason without bumping the version number.

Reply:
I've been setting the `ref` in the policies mix.exs deps. I haven't had any problems with that. Also, when I'm testing policies against changes in open_api_spex, I temporarily set `path: "../open_api_spex" mix.exs deps.

Reply:
Yeah I've been having a lot of problems with `path` and `branch` not updating the lib within the application. I at one point had a `raise` in open api spex and it wasn't showing in the applicatio (even thought it was hosing the open api spex tests). Not sure what's going on but it's frustrating...